var searchData=
[
  ['ufunc_0',['ufunc',['../structufunc.html',1,'']]],
  ['ufunctypeerror_1',['UFuncTypeError',['../classnumpy_1_1__core_1_1__exceptions_1_1UFuncTypeError.html',1,'numpy::_core::_exceptions']]],
  ['uint16dtype_2',['UInt16Dtype',['../classpandas_1_1core_1_1arrays_1_1integer_1_1UInt16Dtype.html',1,'pandas::core::arrays::integer']]],
  ['uint32dtype_3',['UInt32Dtype',['../classpandas_1_1core_1_1arrays_1_1integer_1_1UInt32Dtype.html',1,'pandas::core::arrays::integer']]],
  ['uint64dtype_4',['UInt64Dtype',['../classpandas_1_1core_1_1arrays_1_1integer_1_1UInt64Dtype.html',1,'pandas::core::arrays::integer']]],
  ['uint8dtype_5',['UInt8Dtype',['../classpandas_1_1core_1_1arrays_1_1integer_1_1UInt8Dtype.html',1,'pandas::core::arrays::integer']]],
  ['unaccent_6',['Unaccent',['../classdjango_1_1contrib_1_1postgres_1_1lookups_1_1Unaccent.html',1,'django::contrib::postgres::lookups']]],
  ['unaccentextension_7',['UnaccentExtension',['../classdjango_1_1contrib_1_1postgres_1_1operations_1_1UnaccentExtension.html',1,'django::contrib::postgres::operations']]],
  ['unaryop_8',['unaryop',['../classpandas_1_1core_1_1computation_1_1ops_1_1UnaryOp.html',1,'pandas.core.computation.ops.UnaryOp'],['../classpandas_1_1core_1_1computation_1_1pytables_1_1UnaryOp.html',1,'pandas.core.computation.pytables.UnaryOp']]],
  ['unarypredicate_9',['UnaryPredicate',['../classdjango_1_1contrib_1_1gis_1_1geos_1_1prototypes_1_1predicates_1_1UnaryPredicate.html',1,'django::contrib::gis::geos::prototypes::predicates']]],
  ['unclosingtextiowrapper_10',['UnclosingTextIOWrapper',['../classpip_1_1__vendor_1_1pygments_1_1util_1_1UnclosingTextIOWrapper.html',1,'pip::_vendor::pygments::util']]],
  ['undefinedcomparison_11',['UndefinedComparison',['../classpip_1_1__vendor_1_1packaging_1_1markers_1_1UndefinedComparison.html',1,'pip::_vendor::packaging::markers']]],
  ['undefinedenvironmentname_12',['UndefinedEnvironmentName',['../classpip_1_1__vendor_1_1packaging_1_1markers_1_1UndefinedEnvironmentName.html',1,'pip::_vendor::packaging::markers']]],
  ['undefinedvariableerror_13',['UndefinedVariableError',['../classpandas_1_1errors_1_1UndefinedVariableError.html',1,'pandas::errors']]],
  ['unicodeusernamevalidator_14',['UnicodeUsernameValidator',['../classdjango_1_1contrib_1_1auth_1_1validators_1_1UnicodeUsernameValidator.html',1,'django::contrib::auth::validators']]],
  ['uninstallcommand_15',['UninstallCommand',['../classpip_1_1__internal_1_1commands_1_1uninstall_1_1UninstallCommand.html',1,'pip::_internal::commands::uninstall']]],
  ['uninstallmissingrecord_16',['UninstallMissingRecord',['../classpip_1_1__internal_1_1exceptions_1_1UninstallMissingRecord.html',1,'pip::_internal::exceptions']]],
  ['uninstallpathset_17',['UninstallPathSet',['../classpip_1_1__internal_1_1req_1_1req__uninstall_1_1UninstallPathSet.html',1,'pip::_internal::req::req_uninstall']]],
  ['uninstallpthentries_18',['UninstallPthEntries',['../classpip_1_1__internal_1_1req_1_1req__uninstall_1_1UninstallPthEntries.html',1,'pip::_internal::req::req_uninstall']]],
  ['union_19',['union',['../classdjango_1_1contrib_1_1gis_1_1db_1_1models_1_1aggregates_1_1Union.html',1,'django.contrib.gis.db.models.aggregates.Union'],['../classdjango_1_1contrib_1_1gis_1_1db_1_1models_1_1functions_1_1Union.html',1,'django.contrib.gis.db.models.functions.Union']]],
  ['uniqueallresult_20',['UniqueAllResult',['../classnumpy_1_1lib_1_1__arraysetops__impl_1_1UniqueAllResult.html',1,'numpy::lib::_arraysetops_impl']]],
  ['uniqueconstraint_21',['UniqueConstraint',['../classdjango_1_1db_1_1models_1_1constraints_1_1UniqueConstraint.html',1,'django::db::models::constraints']]],
  ['uniquecountsresult_22',['UniqueCountsResult',['../classnumpy_1_1lib_1_1__arraysetops__impl_1_1UniqueCountsResult.html',1,'numpy::lib::_arraysetops_impl']]],
  ['uniquefordatevalidator_23',['UniqueForDateValidator',['../classrest__framework_1_1validators_1_1UniqueForDateValidator.html',1,'rest_framework::validators']]],
  ['uniqueformonthvalidator_24',['UniqueForMonthValidator',['../classrest__framework_1_1validators_1_1UniqueForMonthValidator.html',1,'rest_framework::validators']]],
  ['uniqueforyearvalidator_25',['UniqueForYearValidator',['../classrest__framework_1_1validators_1_1UniqueForYearValidator.html',1,'rest_framework::validators']]],
  ['uniqueinverseresult_26',['UniqueInverseResult',['../classnumpy_1_1lib_1_1__arraysetops__impl_1_1UniqueInverseResult.html',1,'numpy::lib::_arraysetops_impl']]],
  ['uniquetogethervalidator_27',['UniqueTogetherValidator',['../classrest__framework_1_1validators_1_1UniqueTogetherValidator.html',1,'rest_framework::validators']]],
  ['uniquevalidator_28',['UniqueValidator',['../classrest__framework_1_1validators_1_1UniqueValidator.html',1,'rest_framework::validators']]],
  ['unix_29',['Unix',['../classpip_1_1__vendor_1_1platformdirs_1_1unix_1_1Unix.html',1,'pip::_vendor::platformdirs::unix']]],
  ['unknownextra_30',['UnknownExtra',['../classpip_1_1__vendor_1_1pkg__resources_1_1UnknownExtra.html',1,'pip::_vendor::pkg_resources']]],
  ['unknowntimezoneerror_31',['UnknownTimeZoneError',['../classpytz_1_1exceptions_1_1UnknownTimeZoneError.html',1,'pytz::exceptions']]],
  ['unknowntimezonewarning_32',['UnknownTimezoneWarning',['../classdateutil_1_1parser_1_1__parser_1_1UnknownTimezoneWarning.html',1,'dateutil::parser::_parser']]],
  ['unorderedobjectlistwarning_33',['UnorderedObjectListWarning',['../classdjango_1_1core_1_1paginator_1_1UnorderedObjectListWarning.html',1,'django::core::paginator']]],
  ['unpacker_34',['Unpacker',['../classpip_1_1__vendor_1_1msgpack_1_1fallback_1_1Unpacker.html',1,'pip::_vendor::msgpack::fallback']]],
  ['unpackexception_35',['UnpackException',['../classpip_1_1__vendor_1_1msgpack_1_1exceptions_1_1UnpackException.html',1,'pip::_vendor::msgpack::exceptions']]],
  ['unpickler_36',['Unpickler',['../classpandas_1_1compat_1_1pickle__compat_1_1Unpickler.html',1,'pandas::compat::pickle_compat']]],
  ['unreadableposterror_37',['UnreadablePostError',['../classdjango_1_1http_1_1request_1_1UnreadablePostError.html',1,'django::http::request']]],
  ['unrecognizedarchiveformat_38',['UnrecognizedArchiveFormat',['../classdjango_1_1utils_1_1archive_1_1UnrecognizedArchiveFormat.html',1,'django::utils::archive']]],
  ['unrewindablebodyerror_39',['unrewindablebodyerror',['../classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1UnrewindableBodyError.html',1,'pip._vendor.urllib3.exceptions.UnrewindableBodyError'],['../classpip_1_1__vendor_1_1requests_1_1exceptions_1_1UnrewindableBodyError.html',1,'pip._vendor.requests.exceptions.UnrewindableBodyError']]],
  ['unsatisfiablerequirement_40',['UnsatisfiableRequirement',['../classpip_1_1__internal_1_1resolution_1_1resolvelib_1_1requirements_1_1UnsatisfiableRequirement.html',1,'pip::_internal::resolution::resolvelib::requirements']]],
  ['unserializablecontenterror_41',['UnserializableContentError',['../classdjango_1_1utils_1_1xmlutils_1_1UnserializableContentError.html',1,'django::utils::xmlutils']]],
  ['unsortedindexerror_42',['UnsortedIndexError',['../classpandas_1_1errors_1_1UnsortedIndexError.html',1,'pandas::errors']]],
  ['unsupportedextension_43',['UnsupportedExtension',['../classpip_1_1__vendor_1_1urllib3_1_1contrib_1_1pyopenssl_1_1UnsupportedExtension.html',1,'pip::_vendor::urllib3::contrib::pyopenssl']]],
  ['unsupportedfunctioncall_44',['UnsupportedFunctionCall',['../classpandas_1_1errors_1_1UnsupportedFunctionCall.html',1,'pandas::errors']]],
  ['unsupportedmediatype_45',['UnsupportedMediaType',['../classrest__framework_1_1exceptions_1_1UnsupportedMediaType.html',1,'rest_framework::exceptions']]],
  ['unsupportedoperation_46',['UnsupportedOperation',['../classpip_1_1__vendor_1_1pyproject__hooks_1_1__impl_1_1UnsupportedOperation.html',1,'pip::_vendor::pyproject_hooks::_impl']]],
  ['unsupportedpythonversion_47',['UnsupportedPythonVersion',['../classpip_1_1__internal_1_1exceptions_1_1UnsupportedPythonVersion.html',1,'pip::_internal::exceptions']]],
  ['unsupportedversionerror_48',['UnsupportedVersionError',['../classpip_1_1__vendor_1_1distlib_1_1version_1_1UnsupportedVersionError.html',1,'pip::_vendor::distlib::version']]],
  ['unsupportedwheel_49',['UnsupportedWheel',['../classpip_1_1__internal_1_1exceptions_1_1UnsupportedWheel.html',1,'pip::_internal::exceptions']]],
  ['updateapiview_50',['UpdateAPIView',['../classrest__framework_1_1generics_1_1UpdateAPIView.html',1,'rest_framework::generics']]],
  ['updatecachemiddleware_51',['UpdateCacheMiddleware',['../classdjango_1_1middleware_1_1cache_1_1UpdateCacheMiddleware.html',1,'django::middleware::cache']]],
  ['updateerror_52',['UpdateError',['../classdjango_1_1contrib_1_1sessions_1_1backends_1_1base_1_1UpdateError.html',1,'django::contrib::sessions::backends::base']]],
  ['updatemodelmixin_53',['UpdateModelMixin',['../classrest__framework_1_1mixins_1_1UpdateModelMixin.html',1,'rest_framework::mixins']]],
  ['updatequery_54',['UpdateQuery',['../classdjango_1_1db_1_1models_1_1sql_1_1subqueries_1_1UpdateQuery.html',1,'django::db::models::sql::subqueries']]],
  ['updateview_55',['UpdateView',['../classdjango_1_1views_1_1generic_1_1edit_1_1UpdateView.html',1,'django::views::generic::edit']]],
  ['updatingdefaultshelpformatter_56',['UpdatingDefaultsHelpFormatter',['../classpip_1_1__internal_1_1cli_1_1parser_1_1UpdatingDefaultsHelpFormatter.html',1,'pip::_internal::cli::parser']]],
  ['upgradeprompt_57',['UpgradePrompt',['../classpip_1_1__internal_1_1self__outdated__check_1_1UpgradePrompt.html',1,'pip::_internal::self_outdated_check']]],
  ['uploadedfile_58',['UploadedFile',['../classdjango_1_1core_1_1files_1_1uploadedfile_1_1UploadedFile.html',1,'django::core::files::uploadedfile']]],
  ['uploadfileexception_59',['UploadFileException',['../classdjango_1_1core_1_1files_1_1uploadhandler_1_1UploadFileException.html',1,'django::core::files::uploadhandler']]],
  ['upper_60',['Upper',['../classdjango_1_1db_1_1models_1_1functions_1_1text_1_1Upper.html',1,'django::db::models::functions::text']]],
  ['upperinclusive_61',['UpperInclusive',['../classdjango_1_1contrib_1_1postgres_1_1fields_1_1ranges_1_1UpperInclusive.html',1,'django::contrib::postgres::fields::ranges']]],
  ['upperinfinite_62',['UpperInfinite',['../classdjango_1_1contrib_1_1postgres_1_1fields_1_1ranges_1_1UpperInfinite.html',1,'django::contrib::postgres::fields::ranges']]],
  ['url_63',['Url',['../classpip_1_1__vendor_1_1urllib3_1_1util_1_1url_1_1Url.html',1,'pip::_vendor::urllib3::util::url']]],
  ['urlfield_64',['urlfield',['../classdjango_1_1db_1_1models_1_1fields_1_1URLField.html',1,'django.db.models.fields.URLField'],['../classdjango_1_1forms_1_1fields_1_1URLField.html',1,'django.forms.fields.URLField'],['../classrest__framework_1_1fields_1_1URLField.html',1,'rest_framework.fields.URLField']]],
  ['urlinput_65',['URLInput',['../classdjango_1_1forms_1_1widgets_1_1URLInput.html',1,'django::forms::widgets']]],
  ['urlizer_66',['Urlizer',['../classdjango_1_1utils_1_1html_1_1Urlizer.html',1,'django::utils::html']]],
  ['urlnode_67',['URLNode',['../classdjango_1_1template_1_1defaulttags_1_1URLNode.html',1,'django::template::defaulttags']]],
  ['urlpathversioning_68',['URLPathVersioning',['../classrest__framework_1_1versioning_1_1URLPathVersioning.html',1,'rest_framework::versioning']]],
  ['urlpattern_69',['URLPattern',['../classdjango_1_1urls_1_1resolvers_1_1URLPattern.html',1,'django::urls::resolvers']]],
  ['urlpatternstestcase_70',['URLPatternsTestCase',['../classrest__framework_1_1test_1_1URLPatternsTestCase.html',1,'rest_framework::test']]],
  ['urlrequired_71',['URLRequired',['../classpip_1_1__vendor_1_1requests_1_1exceptions_1_1URLRequired.html',1,'pip::_vendor::requests::exceptions']]],
  ['urlresolver_72',['URLResolver',['../classdjango_1_1urls_1_1resolvers_1_1URLResolver.html',1,'django::urls::resolvers']]],
  ['urlschemeunknown_73',['URLSchemeUnknown',['../classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1URLSchemeUnknown.html',1,'pip::_vendor::urllib3::exceptions']]],
  ['urlvalidator_74',['URLValidator',['../classdjango_1_1core_1_1validators_1_1URLValidator.html',1,'django::core::validators']]],
  ['user_75',['User',['../classdjango_1_1contrib_1_1auth_1_1models_1_1User.html',1,'django::contrib::auth::models']]],
  ['useradmin_76',['UserAdmin',['../classdjango_1_1contrib_1_1auth_1_1admin_1_1UserAdmin.html',1,'django::contrib::auth::admin']]],
  ['userattributesimilarityvalidator_77',['UserAttributeSimilarityValidator',['../classdjango_1_1contrib_1_1auth_1_1password__validation_1_1UserAttributeSimilarityValidator.html',1,'django::contrib::auth::password_validation']]],
  ['userchangeform_78',['UserChangeForm',['../classdjango_1_1contrib_1_1auth_1_1forms_1_1UserChangeForm.html',1,'django::contrib::auth::forms']]],
  ['usercreationform_79',['UserCreationForm',['../classdjango_1_1contrib_1_1auth_1_1forms_1_1UserCreationForm.html',1,'django::contrib::auth::forms']]],
  ['userinstallationinvalid_80',['UserInstallationInvalid',['../classpip_1_1__internal_1_1exceptions_1_1UserInstallationInvalid.html',1,'pip::_internal::exceptions']]],
  ['usermanager_81',['UserManager',['../classdjango_1_1contrib_1_1auth_1_1models_1_1UserManager.html',1,'django::contrib::auth::models']]],
  ['usernamefield_82',['UsernameField',['../classdjango_1_1contrib_1_1auth_1_1forms_1_1UsernameField.html',1,'django::contrib::auth::forms']]],
  ['userpassestestmixin_83',['UserPassesTestMixin',['../classdjango_1_1contrib_1_1auth_1_1mixins_1_1UserPassesTestMixin.html',1,'django::contrib::auth::mixins']]],
  ['userratethrottle_84',['UserRateThrottle',['../classrest__framework_1_1throttling_1_1UserRateThrottle.html',1,'rest_framework::throttling']]],
  ['userserializer_85',['UserSerializer',['../classtutorial_1_1quickstart_1_1serializers_1_1UserSerializer.html',1,'tutorial::quickstart::serializers']]],
  ['usersettingsholder_86',['UserSettingsHolder',['../classdjango_1_1conf_1_1UserSettingsHolder.html',1,'django::conf']]],
  ['userviewset_87',['UserViewSet',['../classtutorial_1_1quickstart_1_1views_1_1UserViewSet.html',1,'tutorial::quickstart::views']]],
  ['usfederalholidaycalendar_88',['USFederalHolidayCalendar',['../classpandas_1_1tseries_1_1holiday_1_1USFederalHolidayCalendar.html',1,'pandas::tseries::holiday']]],
  ['ustimezone_89',['USTimeZone',['../classpytz_1_1reference_1_1USTimeZone.html',1,'pytz::reference']]],
  ['utc_90',['UTC',['../classpytz_1_1UTC.html',1,'pytz']]],
  ['uuidcontains_91',['UUIDContains',['../classdjango_1_1db_1_1models_1_1lookups_1_1UUIDContains.html',1,'django::db::models::lookups']]],
  ['uuidconverter_92',['UUIDConverter',['../classdjango_1_1urls_1_1converters_1_1UUIDConverter.html',1,'django::urls::converters']]],
  ['uuidendswith_93',['UUIDEndsWith',['../classdjango_1_1db_1_1models_1_1lookups_1_1UUIDEndsWith.html',1,'django::db::models::lookups']]],
  ['uuidfield_94',['uuidfield',['../classdjango_1_1db_1_1models_1_1fields_1_1UUIDField.html',1,'django.db.models.fields.UUIDField'],['../classdjango_1_1forms_1_1fields_1_1UUIDField.html',1,'django.forms.fields.UUIDField'],['../classrest__framework_1_1fields_1_1UUIDField.html',1,'rest_framework.fields.UUIDField']]],
  ['uuidicontains_95',['UUIDIContains',['../classdjango_1_1db_1_1models_1_1lookups_1_1UUIDIContains.html',1,'django::db::models::lookups']]],
  ['uuidiendswith_96',['UUIDIEndsWith',['../classdjango_1_1db_1_1models_1_1lookups_1_1UUIDIEndsWith.html',1,'django::db::models::lookups']]],
  ['uuidiexact_97',['UUIDIExact',['../classdjango_1_1db_1_1models_1_1lookups_1_1UUIDIExact.html',1,'django::db::models::lookups']]],
  ['uuidistartswith_98',['UUIDIStartsWith',['../classdjango_1_1db_1_1models_1_1lookups_1_1UUIDIStartsWith.html',1,'django::db::models::lookups']]],
  ['uuidserializer_99',['UUIDSerializer',['../classdjango_1_1db_1_1migrations_1_1serializer_1_1UUIDSerializer.html',1,'django::db::migrations::serializer']]],
  ['uuidstartswith_100',['UUIDStartsWith',['../classdjango_1_1db_1_1models_1_1lookups_1_1UUIDStartsWith.html',1,'django::db::models::lookups']]],
  ['uuidtextmixin_101',['UUIDTextMixin',['../classdjango_1_1db_1_1models_1_1lookups_1_1UUIDTextMixin.html',1,'django::db::models::lookups']]]
];
